lte/gateway/python/integ_tests/s1aptests/test_attach_with_mme_restart_after_auth.py

```diff
@@ -0,0 +1,123 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+import unittest
+
+import orc8r.protos.metricsd_pb2 as metricsd
+import s1ap_types
+import s1ap_wrapper
+from s1ap_utils import MagmadUtil
+
+class TestAttachWithMmeRestartAfterAuth(unittest.TestCase):
```

Comment:
maybe better to name this test as after `...sec_mod_cmd`.

Reply:
Sure, it was the best location to trigger the mme restart after auth was completed due to difficulty of synchronizing the restart